docs/src/verification.md

```diff
@@ -0,0 +1,166 @@
+# Verification in tests
+
+## General Overview
+
+When comparing our `compiled_model` with the `framework_model` (e.g., `PyTorch`), we aim to verify whether the output from the `compiled_model` is sufficiently similar to the output from the `framework_model` (where required degree of similarity is configurable).
```

Comment:
For the introduction section, let's use common language instead of code-specific names (e.g. instead of `compiled_model`, let's use compiled model). 

Using code-specific names is beneficial below examples. In other case, I would more prefer to have common language :))